assets/scripts/main.js

```diff
@@ -30,6 +30,10 @@ import './dialogs/_save_as_image'
 
 // Street
 import './streets/name'
+import './streets/scroll'
+
+// Gallery
+import './gallery/scroll'
```

Comment:
There is `app/focus.js` missing here and there are few side effects defined (`window.addEventListener`).

I know that there are few symbols imported in `exports.js` and I believe these side effects will be executed there, but we intend to remove `exports.js` after migration. BTW do we know, that executing these side effects twice will not hurt us?